### PR TITLE
Improve error message for missing build products

### PR DESCRIPTION
**Pull request by mattludwigs**

### CODE REVIEW

File: priv/templates/script.upload.eex [hunk 1]
```diff
@@ -60,7 +60,20 @@ if [ -z "$FILENAME" ]; then
...
+                echo "MIX_TARGET:    ${MIX_TARGET}"
+                echo "MIX_ENV:       ${MIX_ENV}"
+                echo
+                echo "Make sure your Nerves environment is correct"
+                echo
+                echo "If the Nerves environment is correct make sure you have built the firmware using 'mix firmware'"
+                echo
+                echo "If you are uploading a .fw file from a custom path you can specific the path like so:"
+                echo
+                echo "  ./upload device-destination /path/to/my/frimware.fw"
```
Comment (fhunleth) [inline priv/templates/script.upload.eex:75]:
```suggestion
                echo "  ./upload.sh device-destination /path/to/my/firmware.fw"
```

File: priv/templates/script.upload.eex [hunk 2]
```diff
@@ -60,7 +60,20 @@ if [ -z "$FILENAME" ]; then
...
+                echo "MIX_TARGET:    ${MIX_TARGET}"
+                echo "MIX_ENV:       ${MIX_ENV}"
+                echo
+                echo "Make sure your Nerves environment is correct."
+                echo
+                echo "If the Nerves environment is correct make sure you have built the firmware using 'mix firmware'."
+                echo
+                echo "If you are uploading a .fw file from a custom path you can specify the path like so:"
+                echo
+                echo "  $0 device-destination /path/to/my/firmware.fw"
```
Comment (fhunleth) [inline priv/templates/script.upload.eex:75]:
```suggestion
                echo "  $0 <device hostname or IP address> </path/to/my/firmware.fw>"
```
I'm thinking that `device hostname or IP address` is more obvious than `device-destination`. The last change was to put brackets to show that you need to fill in these fields.

Also, have you tested this to see how everything looks after all of the changes? Like we didn't mess up line wraps or anything like that?

**Comment by mattludwigs** [comment]
@fhunleth here is are screenshots of the output:

image

image

The first is on a smaller terminal window and the second is on a larger window, I can adjust the wording/format to avoid line wraps?

**Comment by fhunleth** [comment]
Thanks!

**Pull request merged**